src/diffusers/configuration_utils.py

```diff
@@ -347,6 +347,7 @@ def load_config(
         _ = kwargs.pop("mirror", None)
         subfolder = kwargs.pop("subfolder", None)
         user_agent = kwargs.pop("user_agent", {})
+        dduf_reader = kwargs.pop("dduf_reader", None)
```

Comment:
`load_config()` is a public method, so I have a few considerations here:

* What happens both `cached_folder` and `dduf_reader` are supplied?
* Should the combination of `cached_folder` and `dduf_reader` values be validated?

Reply:
passing `dduf_reader` or `dduf_entites` means that we already downloaded the files and prepared everything to read the file. So cached_folder is not really useful at this stage I think.